org.eclipse.jdt.ls.tests/.project

```diff
@@ -5,11 +5,6 @@
 	<projects>
 	</projects>
 	<buildSpec>
-		<buildCommand>
-			<name>org.eclipse.jdt.core.javabuilder</name>
-			<arguments>
-			</arguments>
-		</buildCommand>
```

Comment:
Revert the change in this file.

Sometimes when you run the PDE tests in VS Code, the java nature will be removed with some reason, but I have no idea why and who did it.

Reply:
Shoot did not mean to push these changes, will fix!